.github/workflows/CD.yml

```
@@ -0,0 +1,91 @@
+      run: |
+        mkdir -p $CREATE_SECRET_DIR
+        echo -n "$CREATE_SECRET" | base64 --decode > $CREATE_SECRET_DIR/$CREATE_SECRET_DIR_FILE_NAME
+
+    - name: 5. Grant execute permission for gradlew
+      run: chmod +x gradlew
+
+    - name: 6. Build with Gradle, without test
+      run: ./gradlew clean build -x test
+      
+    - name: 7. Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)